packages/@stimulus/core/src/string_helpers.ts

```diff
@@ -9,3 +9,7 @@ export function capitalize(value: string) {
 export function dasherize(value: string) {
   return value.replace(/([A-Z])/g, (_, char) => `-${char.toLowerCase()}`)
 }
+
+export function tokenize(value: string) {
+  return value.trim().split(/\s+/).filter(content => content.length)
```

Comment:
A simpler (and slightly faster) method of doing this is
```javascript
return value.match(/[^ ]+/g)
```

Reply:
Please do add a PR 👍

Reply:
PR https://github.com/hotwired/stimulus/pull/430